student-store-ui/src/components/App/App.jsx

```diff
@@ -1,21 +1,114 @@
 import * as React from "react"
-import { BrowserRouter } from 'react-router-dom'
+import axios from "axios"
+import { useEffect, useState } from "react"
+
+import { BrowserRouter, Route, Routes, useParams } from 'react-router-dom'
 import Navbar from "../Navbar/Navbar"
 import Sidebar from "../Sidebar/Sidebar"
 import Home from "../Home/Home"
 import "./App.css"
+import Hero from "../Hero/Hero"
+import Category from "../Categories/Categories"
+import AboutUs from "../AboutUs/AboutUs"
+import ContactUs from "../ContactUs/ContactUs"
 
 export default function App() {
+
+  // const url = "https://codepath-store-api.herokuapp.com/store";
+  const url = "http://localhost:3001/"
+
+  const {id} = useParams();
+  const [products, setProducts] = useState([]);
+  const [originalProducts, setOriginalProducts] = useState([])
+  const [quantities, setQuantities] = useState([])
+  const [cart, setCart] = React.useState(quantities.filter(([id, quantity]) => quantity > 0))
+
```

Comment:
be consistent with how you call `useState()` (vs. `React.useState()` - NOT preferred)